README.md

```diff
@@ -115,13 +115,53 @@ i.e. "${var.environment}-${var.config}-teacherservices-cloud"
 
 e.g. cluster99-development-teacherservices-cloud
 
+#### GlobalSign certificate generation
+
+##### Generate value for DNS record
```

Comment:
These 3 sections should be added to a new section called "Add new top domain"
And mention this has already been done. We don't need to do it again if we add a new cluster.